python/tvm/relay/build_module.py

```diff
@@ -17,7 +17,7 @@
     "FoldConstant": 2,
     "CombineParallelConv2D": 3,
     "FoldScaleAxis": 3,
-    "AlterOpLayout": 3,
+    "AlterOpLayout": 4,
```

Comment:
cc @merrymercy

Reply:
What's wrong with this pass? @ZihengJiang

Reply:
@merrymercy I met some error too.
```
/root/tvm/src/relay/ir/op.cc:63: Operator attribute 'FTVMAlterOpLayout' is not registered
[bt] (0) /root/tvm/build/libtvm.so(dmlc::StackTrace[abi:cxx11](unsigned long)+0x1fd) [0x7f26b24f61cd]
[bt] (1) /root/tvm/build/libtvm.so(dmlc::LogMessageFatal::~LogMessageFatal()+0x2d) [0x7f26b24f6e1d]
[bt] (2) /root/tvm/build/libtvm.so(tvm::relay::Op::GetGenericAttr(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)+0x1e2) [0x7f26b27ea032]
[bt] (3) /root/tvm/build/libtvm.so(tvm::relay::alter_op_layout::CallAlter(tvm::relay::Call const&, std::vector<tvm::relay::Expr, std::allocator<tvm::relay::Expr> > const&)+0x96b) [0x7f26b285b16b]
[bt] (4) /root/tvm/build/libtvm.so(tvm::relay::alter_op_layout::AlterOpLayoutRewrite(tvm::relay::Call const&, tvm::Array<tvm::relay::Expr, void> const&, tvm::NodeRef const&)+0x135d) [0x7f26b285c79d]
```

Reply:
Fixed by https://github.com/dmlc/tvm/pull/2289

Reply:
@ZihengJiang revert this change